src/main/java/frc/robot/commands/conveyor/ToggleConveyorCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands.conveyor;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+
+public class ToggleConveyorCommand extends CommandBase
+{
+    private boolean reversed;
+    
+    public ToggleConveyorCommand(boolean reversed)
```

Comment:
It seems like the command doesn't need to keep tract of reversed, and the conveyor can track the state(direction) already so remove all references in this class to reversed.